core/networks.go

```diff
@@ -94,6 +103,8 @@ func (n Network) MinGenesisTime() uint64 {
 		return 1616508000
 	case HoleskyNetwork:
 		return 1695902400
+	case HoodiNetwork:
+		return 1742212800 + 600
```

Comment:
why not write the final num?

Reply:
@MatusKysel Sorry for unresolving this but it's not clear why it consists of two numbers. Can you please explain it here so that it will be searchable? A comment would be better though but it's already been merged

Reply:
@MatusKysel  thanks